src/style.css

```
@@ -1 +1,80 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('/src/fonts/Avenir-Book.ttf') format('truetype'),
+       url('/src/fonts/Avenir-Heavy.ttf') format('truetype'),
+       url('/src/fonts/Avenir.ttc') format('opentype'),
+}
+
+.body {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  margin-top: 20px;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+}
+
+.search {
+  display: flex;
+  flex-direction: column;
+}
+
+.search__search-bar {
+  font-family: inherit;
+  font-size: inherit;
+  border: 1px solid #E1E7ED;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+  background-image: url(images/Search.svg);
+  color: #3D4E61;
+  border-radius: 4px;
```

```suggestion
  border-radius: 4px;
  outline: none;
```
Consider removing outlines here